src/templates/Cart/index.tsx

```diff
@@ -46,7 +47,14 @@ const Cart = ({
             hasLink
           />
         )}
-
+        <S.Text>
+          <ErrorWarning size={18} /> Your purchase is protected by a secure
```

Comment:
Then here, we could change to use the `Info`

```suggestion
          <Info size={18} /> Your purchase is protected by a secure
```